manifests/init.pp

```diff
@@ -48,6 +48,7 @@
   $sshd_config_allowusers              = [],
   $sshd_config_denygroups              = [],
   $sshd_config_denyusers               = [],
+  $sshd_config_maxauthtries            = 6,
```

Comment:
Sometimes people will inadvertently use a string `'6'`. To deal with this we can do something like

``` puppet
$sshd_config_maxauthtries_real = $sshd_config_maxauthtries + 0
validate_integer($sshd_config_maxauthtries_real)
```